pep-0001.txt

```diff
@@ -793,16 +792,16 @@ References and Footnotes
    https://github.com/python/peps/blob/main/README.rst
 
 .. [3] `CODEOWNERS` documentation
-   (https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
+   (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
```

Comment:
Since this footnote is just the link, may as well make it actual link target instead, to avoid an extra layer of indirection

Reply:
And this? aad94a630a9b2cc07d97efe8669a49962a25eb56

Reply:
Yup, looks good :+1: